x-pack/test/functional/apps/api_keys/home_page.ts

```diff
@@ -173,6 +173,11 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
 
         expect(await browser.getCurrentUrl()).to.contain('app/management/security/api_keys');
 
+        await retry.try(async () => {
+          const submitButton = await pageObjects.apiKeys.getSubmitButtonOnApiKeyFlyout();
+          expect(await submitButton.isEnabled()).to.be(true);
```

Comment:
__question:__ would it be possible to use something like this instead (right here or inside `page_objects/api_keys_page.ts`)?
```ts
await testSubjects.waitForEnabled('formFlyoutSubmitButton', 10000);
```

Reply:
Great suggestion! Updated in latest commit.